front-end/src/components/analysis-page/AnalysisPage.jsx

```diff
@@ -50,6 +50,18 @@ export class AnalysisPage extends Component {
           >
             <div className="flex gap-y-10 w-full flex-col h-full">
               <div className= "w-full bg-gray-100 pb-2">
+                <div
+                  className="absolute m-2 mx-auto px-3" data-testid="arrow-right"
+                >
+                  <BaseButton
```

Comment:
Could we maybe make it so that the icon isn't used in the basebutton? You can look at the title.svg for reference, but I think it would look better if we just imported the svg as a react component then just added the on-click feature to that instead of using the base button, because of the styling on it not looking quite right on the navbar.

Reply:
+1